firmware/config/boards/proteus/board_configuration.cpp

```diff
@@ -128,6 +133,14 @@ static void setupSdCard() {
 	engineConfiguration->spi3mosiPin = Gpio::C12;
 }
 
+static void setupPowerGoodPins() {
```

Comment:
there's already a hook called `initBoardSensors()`, use that

DO NOT call `efiSetPad...` from `setBoardConfigOverrides()`